randovania/data/json_data/prime1.txt

```diff
@@ -2403,582 +2411,1261 @@ Ruins Entrance
+  > Dock to Transport Access North
+      Trivial
 
 > Dock to Transport Access North; Heals? False
   * Normal Door to Transport Access North/Dock to Transport to Magmoor Caverns North
+  > Dock to Sun Tower
+      Trivial
+  > Dock to Vault Access
+      Trivial
+  > Elevator to Magmoor Caverns - Transport to Chozo Ruins North
+      Scan Visor
+
+> Elevator to Magmoor Caverns - Transport to Chozo Ruins North; Heals? False; Spawn Point
+  * Teleporter to Magmoor Caverns - Transport to Chozo Ruins North
+  > Dock to Transport Access North
+      Trivial
 
 ----------------
 Sunchamber Lobby
 Asset id: 1025740749
 > Dock to Arboretum; Heals? False
-  * Normal Door to Arboretum/Dock to Sunchamber Lobby
+  * Missile Blast Shield to Arboretum/Dock to Sunchamber Lobby
+  > Dock to Sunchamber Access
+      Trivial
 
 > Dock to Sunchamber Access; Heals? False
   * Normal Door to Sunchamber Access/Dock to Sunchamber Lobby
+  > Dock to Arboretum
+      Trivial
 
 ----------------
 Gathering Hall Access
 Asset id: 2515665310
 > Dock to Gathering Hall; Heals? False
   * Normal Door to Gathering Hall/Dock to Gathering Hall Access
+  > Dock to Arboretum
+      Trivial
 
 > Dock to Arboretum; Heals? False
-  * Normal Door to Arboretum/Dock to Gathering Hall Access
+  * Missile Blast Shield to Arboretum/Dock to Gathering Hall Access
+  > Dock to Gathering Hall
+      Trivial
 
 ----------------
 Tower Chamber
 Asset id: 297624503
 > Dock to Tower of Light; Heals? False
-  * Normal Door to Tower of Light/Dock to Tower Chamber
+  * Wave Door to Tower of Light/Dock to Tower Chamber
+  > Pickup (Artifact of Lifegiver)
+      Trivial
+
+> Pickup (Artifact of Lifegiver); Heals? False
+  * Pickup 14; Major Location? True
+  > Dock to Tower of Light
+      Trivial
 
 ----------------
 Ruined Gallery
 Asset id: 3813660971
 > Dock to North Atrium; Heals? False
   * Normal Door to North Atrium/Dock to Ruined Gallery
+  > Dock to Totem Access
+      Trivial
+  > Dock to Map Station
+      Trivial
+  > Pickup (Missile Expansion)
+      Missile
+  > Pickup (Missile Expansion 2)
+      Morph Ball Bomb and Morph Ball
 
 > Dock to Totem Access; Heals? False
   * Normal Door to Totem Access/Dock to Ruined Gallery
+  > Dock to North Atrium
+      Trivial
 
 > Dock to Map Station; Heals? False
-  * Normal Door to Map Station/Dock to Ruined Gallery
+  * Missile Blast Shield to Map Station/Dock to Ruined Gallery
+  > Dock to North Atrium
+      Trivial
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 15; Major Location? False
+  > Dock to North Atrium
+      Trivial
+
+> Pickup (Missile Expansion 2); Heals? False
+  * Pickup 16; Major Location? False
+  > Dock to North Atrium
+      Trivial
 
 ----------------
 Sun Tower
 Asset id: 3725988722
 > Dock to Sun Tower Access; Heals? False
   * Normal Door to Sun Tower Access/Dock to Sun Tower
+  > Dock to Transport to Magmoor Caverns North
+      Trivial
```

Comment:
Could add a connection to the ghost event, and the ghost event could connect to the top door as well with requirement, morph and spider

Reply:
Done